resources/assets/lib/models/user.ts

```diff
@@ -92,7 +93,8 @@ export default class User {
 
 const deletedUser = new User(-1);
 deletedUser.isDeleted = true;
-deletedUser.username = osu.trans('users.deleted');
+// deletedUser.username = osu.trans('users.deleted');
```

Comment:
Somehow this line causing webpack error, I don't know why.

![Screenshot from 2021-07-23 15-57-03](https://user-images.githubusercontent.com/4964985/126761592-eb941a08-c3ba-4da9-869b-9cd04ad77340.png)

Reply:
`osu` hasn't been initialized yet when this module is loaded because there's now an implicit circular dependency between `osu-core` and `osu-common` modules.

Reply:
You're right.

`ajaxError` in `osu-common` is calling `osu-core`. Looks like I should move that function to `utils/ajax`.